src/sql/platform/capabilities/common/capabilitiesService.ts

```diff
@@ -22,7 +22,7 @@ export const clientCapabilities = {
 
 export interface ConnectionProviderProperties {
 	providerId: string;
-	iconPath?: URI | IconPath | { id: string, path: IconPath }[]
+	iconPath?: URI | IconPath | { id: string, path: IconPath, default?: boolean }[]
```

Comment:
Would suggest moving this out into its own type (ProviderIcon or something similar) and then update the pathConfigs to use that type as well.